packages/grafana-ui/src/components/Button/ToolbarButton.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { forwardRef, HTMLAttributes } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { styleMixins, useStyles } from '../../themes';
+import { IconName } from '../../types/icon';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { Icon } from '../Icon/Icon';
+
+export interface Props extends HTMLAttributes<HTMLButtonElement> {
+  /** Icon name */
+  icon?: IconName;
+  /** Tooltip */
+  tooltip?: string;
+  /** For image icons */
+  imgSrc?: string;
+  /** if true or false will show angle-down/up */
+  isOpen?: boolean;
+  /** Controls flex-grow: 1 */
+  fullWidth?: boolean;
+  /** reduces padding to xs */
+  narrow?: boolean;
+}
+
+export const ToolbarButton = forwardRef<HTMLButtonElement, Props>(
+  ({ tooltip, icon, className, children, imgSrc, fullWidth, isOpen, narrow, ...rest }, ref) => {
+    const styles = useStyles(getStyles);
+
+    const contentStyles = cx({
+      [styles.content]: true,
+      [styles.contentWithIcon]: !!icon,
+      [styles.contentWithRightIcon]: isOpen !== undefined,
+    });
+
+    const buttonStyles = cx(
+      {
+        [styles.button]: true,
+        [styles.buttonFullWidth]: fullWidth,
+        [styles.narrow]: narrow,
+      },
+      className
+    );
+
+    const body = (
+      <button ref={ref} className={buttonStyles} {...rest}>
+        {icon && <Icon name={icon} size={'lg'} />}
+        {imgSrc && <img className={styles.img} src={imgSrc} />}
+        {children && <span className={contentStyles}>{children}</span>}
+        {isOpen === false && <Icon name="angle-down" />}
+        {isOpen === true && <Icon name="angle-up" />}
```

Comment:
```suggestion
      {isOpen ? <Icon name="angle-up" /> : <Icon name="angle-down" />}
```
I think using a ternary here makes the code cleaner.

Reply:
just find ? less readable in this case